docs/sdk/wrapped-keys/overview.md

```diff
@@ -60,12 +40,146 @@ yarn add @lit-protocol/wrapped-keys
 </TabItem>
 </Tabs>
 
-With the SDK, you have the following functions available to you:
+## How it Works
+
+Currently the available actions of the Wrapped Keys SDK are:
```

Comment:
> 'Currently the available actions of the Wrapped Keys SDK are:'

consider rephrasing to 'the current functionality offered by the Wrapped Keys SDK includes:'